x-pack/plugin/sql/qa/jdbc/security/with-ssl/build.gradle

```diff
@@ -6,3 +8,10 @@ testClusters.all {
   setting 'xpack.security.http.ssl.enabled', 'true'
   setting 'xpack.security.transport.ssl.enabled', 'true'
 }
+
+// JDBC client can only be configured for SSL with keystores, but we can't use JKS/PKCS12 keystores in FIPS 140-2 mode.
+if (BuildParams.inFipsJvm) {
+  project.tasks.configureEach {
+    enabled = true
```

Comment:
I'm confused, shouldn't this be `false`? Also, we don't want to diasble _all_ tasks, just `Test` tasks. Let's do this:

```
tasks.withType(Test).configureEach {
  onlyIf { BuildParams.inFipsJvm == false}
}
```

Reply:
> I'm confused, shouldn't this be false?

Copy paste error. Will address your suggestion and merge